e2e/dashboards-suite/dashboard-time-zone.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Dashboard time zone support', () => {
     e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
   });
 
-  it('Tests dashboard time zone scenarios', () => {
+  it.skip('Tests dashboard time zone scenarios', () => {
```

Comment:
skipping this 4 year old test for now, until we have an alternative that is using `timeseries` instead of `graph`